front/src/components/index.js

```diff
@@ -1,2 +0,0 @@
-export { default as HelloWorldOneFile } from './HelloWorldOneFile.vue';
-export { default as HelloWorld } from './hello-world/hello-world.component.vue';
```

Comment:
it is useful file
before deleting think why it was added
yes, hello-world were removed but it is place to export all components for shortcut import just only from `@components`

Reply:
https://www.freecodecamp.org/news/how-to-use-es6-modules-and-why-theyre-important-a9b20b480773/

Reply:
https://medium.com/groww-engineering/module-aliasing-in-webpack-f02fe1b91f94